src/px-script/main/datepicker/index.js

```diff
@@ -57,6 +57,11 @@ const _createDatepicker = datepicker => {
     return datepickerObj;
 };
 
+const _destroyDatepickers = () => {
+    _datepickers.forEach(datepicker => datepicker.destroy());
```

Comment:
With this you can remove the next line
```suggestion
    _datepickers = _datepickers.filter(datepicker => datepicker.destroy());
```